SwiftCompilerSources/Sources/Basic/Utils.swift

```diff
@@ -64,11 +64,34 @@ public struct StringRef : CustomStringConvertible, NoReflectionChildren {
 
   public var string: String { _bridged.string }
   public var description: String { string }
-  
+
+  public var count: Int {
+#if $NewCxxMethodSafetyHeuristics
+    Int(_bridged.bytes_end() - _bridged.bytes_begin())
+#else
+    Int(_bridged.__bytes_endUnsafe() - _bridged.__bytes_beginUnsafe())
+#endif
+  }
+
+  public subscript(index: Int) -> UInt8 {
+#if $NewCxxMethodSafetyHeuristics
+    let buffer = UnsafeBufferPointer<UInt8>(start: _bridged.bytes_begin(),
+                                            count: count)
+#else
+    let buffer = UnsafeBufferPointer<UInt8>(start: _bridged.bytes_begin(),
+                                            count: count)
```

Comment:
@zoecarver both #if and #else branches seems to have the same code? Did we mean to say `_bridged.__bytes_beginUnsafe()` in the #else case?